### PR TITLE
ANSIENG-1063/1067

### PR DESCRIPTION
**Pull request by utkarsh5474**
# Description

Molecule tests for zookeeper-tls-rhel and zookeeper-mtls-rhel use the incorrect location of all.yml. This PR resolves that. 

Fixes # (ANSIENG-1063 ANSIENG-1067)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

cp-ansible-on-demand run 77 passes for both zookeeper-tls-rhel and zookeeper-mtls-rhel. Logs available on the jenkins job. 

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: molecule/zookeeper-mtls-rhel/molecule.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ provisioner:
     defaults:
       hash_behaviour: merge
   playbooks:
-    converge: ../../../../all.yml
+    converge: ../../playbooks/all.yml
```
[the remark this answers is not in the document]
Comment (utkarsh5474) [reply molecule/zookeeper-mtls-rhel/molecule.yml:149]: @nsharma-git 
Like confluent.platform.all, right?
It doesn't work. I have tried everything from `../../../../../confluent.platform.all` to `confluent.platform.all`
It just needs the actual location of all.yml

[the remark this answers is not in the document]
Comment (utkarsh5474) [reply molecule/zookeeper-mtls-rhel/molecule.yml:149]: Thanks @erikgb 
I have used the process defined here https://github.com/confluentinc/cp-ansible/blob/7.0.1-post/HOW_TO_TEST.md and not `ansible-galaxy collection install \
  git+https://github.com/confluentinc/cp-ansible.git,<branch>`
Also, I wasn't able to see any other molecule.yml file where fqcn was used. Can you please help point out which molecule.yml file is it? 

Internally, yeah this works everywhere else.

[the remark this answers is not in the document]
Comment (nsharma-git) [reply molecule/zookeeper-mtls-rhel/molecule.yml:149]: Quite a few places we call the playbook with collection based nomenclature. i.e https://github.com/confluentinc/cp-ansible/blob/fd56742ff5b63a75d4be64b7c5d4118b68fbe2ee/molecule/rbac-mds-scram-custom-rhel/prepare.yml#L9

The reason you don't see in molecule.yaml, as its taking the default config which calls the confluent.platform.all
I don't see any reason to specify the playbook name explicitly here. Did you try without specifying the playbook at all.

[the remark this answers is not in the document]
Comment (utkarsh5474) [reply molecule/zookeeper-mtls-rhel/molecule.yml]: Yeah, I understand that. I have tested now locally without the playbook name altogether (explicitly removed). It works fine.
I am not sure why it was used in the first place. That's why I didn't remove it earlier.

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**